### DISCUSSION TITLE
Version 1.40.07

### DISCUSSION THREAD

**Discussion by shred86** [shred86/Ortho4XP#38] [Announcements]
## Bug Fixes
* Fixed issue where paths with spaces on Windows was causing issues with 7-Zip (7z).
* Removed unavailable OSM FR and updated RU server with a working URL.

## Known Issue
* (Previous) One minor issue exists for macOS users where if the "Cancel" option is used in the prompt when changing tiles with unsaved changes, sometimes the prompt will assert a second time.

## Download
* See the [Installation section](https://github.com/shred86/Ortho4XP/wiki/Installation) in the [Wiki](https://github.com/shred86/Ortho4XP/wiki) for the latest download links.

<hr /><em>This discussion was created from the release <a href='https://github.com/shred86/Ortho4XP/releases/tag/v1.40.07'>Version 1.40.07</a>.</em>

**Comment by cdtrent** [comment]
For your next release, would it be possible to move the min_angle parameter from the global tab of the setup page to the per-tile tab? 
I vary this parameter for just those tiles which fail mesh creation with the "bad triangles" error - changing this for those few tiles from the individual tile configs would allow me to batch-update more easily. 
Thanks,
Chris

**Reply by shred86** [reply]
The `min_angle` parameter is on both the tile and global config tab. It should be in the same exact spot, under `limit_tris` in the "Mesh" column. 

Or are you asking for a way to only update the `min_angle` parameter on tiles that already have a tile config? The only way to do this right now is to update your global config to match your tile configs you're trying to update, adjust the `min_angle` as desired then do a batch update with "Override tile config" selected.
